tools/cell_census_builder/scripts/release/census_summary.py

```diff
@@ -2,10 +2,12 @@
 
 import cell_census
 import pandas as pd
-from cell_census_builder.globals import CENSUS_DATA_NAME
+
+from tools.cell_census_builder.globals import CENSUS_DATA_NAME, CENSUS_INFO_NAME
 
 if __name__ == "__main__":
     census_version = sys.argv[1] if len(sys.argv) > 1 else "latest"
+    previous_census_version = sys.argv[2] if len(sys.argv) > 2 else None
```

Comment:
Here we could infer the previous version automatically. Does the Census API expose a list of all available versions?

Reply:
You can't infer it.  "previous" is entirely context dependent at this point, and we will need to run this script manually, i.e.,
> generate_diff census_tag_1 census_tag_2

I'm not sure you can ever automate this for all cases, as it is a human who eventually decides what builds are significant.